hngprojects/hng_boilerplate_java_web
# Feature: Comment Functionality

### How should this be manually tested?
* Submit a comment to a blog post without being authenticated.
* Submit a comment with an expired bearer token.
* Submit a comment with an invalid token.
The expected result is to receive a 401 Unauthorized response for each case.
### Checklist of what you did:
* Implemented authentication and authorization checks for the comment endpoint.
* Validated that the token is not expired.
* Ensured that the token is valid.
* Restricted access to the comment endpoint if any validation checks fail.
* Returned an unauthorized error response if validation checks fail.
* Implemented input validation and sanitization for comment data.
* Ensured comments are correctly linked to respective blog posts.
### Reference Issue 
[FEAT]: Comment Functionality - Add Comment
